pkg/managed/clusterroles.go

```diff
@@ -146,100 +141,105 @@ func newClusterRoles(binding *v1beta1v8o.VerrazzanoBinding, clusterName string)
-				Verbs:     []string{"*"},
-			},
-			{
-				NonResourceURLs: []string{"/version/*"},
-				Verbs:           []string{"get"},
-			},
-			{
-				APIGroups: []string{"apps"},
-				Resources: []string{"deployments", "daemonsets", "replicasets", "statefulsets"},
-				Verbs:     []string{"*"},
-			},
-			{
-				APIGroups:     []string{"apps"},
-				Resources:     []string{"deployments/finalizers"},
-				ResourceNames: []string{"coherence-operator"},
-				Verbs:         []string{"update"},
-			},
-			{
-				APIGroups: []string{"extensions"},
-				Resources: []string{"daemonsets", "replicasets", "statefulsets", "podsecuritypolicies"},
-				Verbs:     []string{"get", "list", "watch", "create", "update", "delete"},
-			},
-			{
-				APIGroups: []string{"policy"},
-				Resources: []string{"podsecuritypolicies"},
-				Verbs:     []string{"get"},
-			},
-			{
-				APIGroups: []string{"batch"},
-				Resources: []string{"jobs"},
-				Verbs:     []string{"get", "list", "watch", "create", "update", "delete", "patch", "deletecollection"},
-			},
-			{
-				APIGroups: []string{"authentication.k8s.io"},
-				Resources: []string{"tokenreviews"},
-				Verbs:     []string{"create"},
-			},
-			{
-				APIGroups: []string{"authorization.k8s.io"},
-				Resources: []string{"subjectaccessreviews"},
-				Verbs:     []string{"create"},
-			},
-			{
-				APIGroups: []string{"apiextensions.k8s.io"},
-				Resources: []string{"customresourcedefinitions"},
-				Verbs:     []string{"*"},
-			},
-			{
-				APIGroups: []string{"monitoring.coreos.com"},
-				Resources: []string{"servicemonitors"},
-				Verbs:     []string{"get", "create"},
-			},
-			{
-				APIGroups: []string{"verrazzano.io"},
-				Resources: []string{"*"},
-				Verbs:     []string{"get", "list", "watch", "create", "update", "delete"},
-			},
-			{
-				APIGroups: []string{"rbac.authorization.k8s.io"},
-				Resources: []string{"clusterroles", "roles"},
-				Verbs:     []string{"get", "list", "watch", "create", "update", "delete"},
-			},
-			{
-				APIGroups: []string{"rbac.authorization.k8s.io"},
-				Resources: []string{"clusterrolebindings", "rolebindings"},
-				Verbs:     []string{"get", "list", "watch", "create", "update", "delete", "patch"},
-			},
-			{
-				APIGroups: []string{"weblogic.oracle"},
-				Resources: []string{"domains"},
-				Verbs:     []string{"get", "list", "watch", "create", "update", "delete", "patch", "deletecollection"},
-			},
-			{
-				APIGroups: []string{"weblogic.oracle"},
-				Resources: []string{"domains/status"},
-				Verbs:     []string{"get", "list", "watch", "update", "patch"},
-			},
-			{
-				APIGroups: []string{"coherence.oracle.com"},
-				Resources: []string{"*"},
-				Verbs:     []string{"*"},
-			},
-		},
+		Rules: getRules(),
 	}
 	clusterRoles = append(clusterRoles, clusterRole)
 
 	return clusterRoles
 }
+
+func getRules() []rbacv1.PolicyRule {
```

Comment:
why is this a function? what's wrong with a constant?

Reply:
ok - i tried it, and now i see - slices are mutable so cannot be assigned to a const. i guess it could be a var though...

Reply:
changed it to a var